assets/src/scripts/models.js

```diff
@@ -48,19 +48,23 @@ export function getTimeseries({sites, params=['00060'], startDate=null, endDate=
                         let startDate = new Date(series.values[0].value[0].dateTime);
                         let endDate = new Date(
                             series.values[0].value.slice(-1)[0].dateTime);
+                        let noDataValue = series.variable.noDataValue;
                         return {
                             code: series.variable.variableCode[0].value,
                             variableName: series.variable.variableName,
                             variableDescription: series.variable.variableDescription,
                             seriesStartDate: startDate,
                             seriesEndDate: endDate,
-                            values: series.values[0].value.map(value => {
-                                let date = new Date(value.dateTime);
+                            values: series.values[0].value.map(datum => {
+                                let date = new Date(datum.dateTime);
+                                let value = parseFloat(datum.value);
                                 return {
                                     time: date,
-                                    value: parseFloat(value.value),
-                                    label: `${formatTime(
-                                        date)}\n${value.value} ${series.variable.unit.unitCode}`
+                                    value: value !== noDataValue ? parseFloat(datum.value) : null,
+                                    qualifiers: datum.qualifiers,
+                                    approved: datum.qualifiers.indexOf('A') > -1,
+                                    estimated: datum.qualifiers.indexOf('E') > -1,
+                                    label: `${formatTime(date)}\n${datum.value} ${series.variable.unit.unitCode} (Qualifiers: ${datum.qualifiers.join(', ')})`
```

Comment:
As I understand from a conversation with Brad, "approved" has the "A" qualifier, and pending has "P". Rather than test for "P", this just checks for A, as one or the other should always be set.